copyweb.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+"""Copy a website into Freenet -- either a single page or the full site.
+
+A bridge between wget and pyFreenet."""
+
+import argparse
+import fcp
+import os
+import sys
+import subprocess
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument("pages", help="the URL to the website to copy", nargs="+")
+parser.add_argument("-d", "--target-directory",
+                    default=None,
+                    help="target directory (default: host part of the path)")
+parser.add_argument("--mirror", action="store_true",
+                    default=False,
+                    help="Mirror the whole site (default: only the pages explicitly given on the commandline)")
+
+args = parser.parse_args()
+
+wget_program = "wget"
+
+wget_mode_option_lists = {
+    "mirror": ["-m", "-nc", "-N", "-k", "-p", "-np", "-nH", "-nd", "-E", "--no-check-certificate", "-e", "robots=off", "-U", "'Mozilla/5.0 (X11; U; Linux i686; en-US; rv:1.8.1.6) Gecko/20070802 SeaMonkey/1.1.4'"],
+    "single_page": ["-t", "2", "-np", "-N", "-k", "-p", "-nd", "-nH", "-H", "-E", "--no-check-certificate", "-e", "robots=off", "-U", "'Mozilla/5.0 (X11; U; Linux i686; en-US; rv:1.8.1.6) Gecko/20070802 SeaMonkey/1.1.4'"],
+}
+
+wget_options = {
+    "target_directory": "--directory-prefix={}"
+}
+
+command = [wget_program]
+if args.mirror:
+    command.extend(wget_mode_option_lists["mirror"])
+else:
+    command.extend(wget_mode_option_lists["single_page"])
+
+if args.target_directory:
```

Comment:
Instead of having interactive usage like this what about making the directory a required argument?